src/inputs/dnstap/DnstapInputStream.cpp

```diff
@@ -224,6 +240,7 @@ void DnstapInputStream::_create_frame_stream_tcp_socket()
 
     // spawn the loop
     _io_thread = std::make_unique<std::thread>([this] {
+        _timer->start(uvw::TimerHandle::Time{1000}, uvw::TimerHandle::Time{RUNNING_NOTIFY_INTERVAL * 1000});
```

Comment:
is there a spot where we need to `stop()` or `close()` or something, when the input is stopped (not just on error event?)

Reply:
based on their unit tests it seems we dont actually need to stop and close it. but I did it on async event as well